### DISCUSSION TITLE
podman-compose example

### DISCUSSION THREAD

**Discussion by heapxor** [dani-garcia/vaultwarden#4277] [Q&A]
helo,
i saw the article about daemonization/using systemd + podman (https://github.com/dani-garcia/vaultwarden/wiki/Using-Podman) 
but anyone had a chance to prepare conf yaml for the podman-compose ie. to have it all in declarative configuration. 

thanks

**Comment by septatrix** [comment]
The systemd file is already declarative and I would recommend it (or podman-quart - which also uses systemd files) over podman-compose as the former ties it better into the service and lifecycle management on Linux.
If you really want a compose file you should be able to just use the [docker-compose file](https://github.com/dani-garcia/vaultwarden/wiki/Using-Docker-Compose) as it only uses basic features and both tools are compatible unless one uses some obscure features.

**Reply by heapxor** [reply]
@septatrix thanks i will check,
what do u mean by podman-quart? cant google it

i found only
https://www.redhat.com/sysadmin/quadlet-podman
https://quarkus.io/guides/quartz


thanks

**Reply by septatrix** [reply]
I misremembered the name, quadlet is the correct name. It was once a separate project but is not included in podman as [`man:podman-systemd.unit`](https://docs.podman.io/en/latest/markdown/podman-systemd.unit.5.html)
